hazelcast-jet-core/src/test/java/com/hazelcast/jet/config/ResourceConfigTest.java

```diff
@@ -87,6 +87,16 @@ public void when_addResourcesWithPackage() {
                 ));
     }
 
+    @Test
+    public void when_addResourcesWithNonExistingPackage() {
+        // When
+        config.addPackage("this.package.is.not.exist");
```

Comment:
this.package.is.not.exist -> this.package.does.not.exist ?

Reply:
make sense, I'll also change it to valid package name, so I'll use `thispackage.does.not.exist`